### PR TITLE
Added CoPresent icon from google material design

### LINKED ISSUE #510: Instructor Insights Icon for Course Features
### Steps to Reproduce

The OCW team is adding the "Instructor Insights" tag to the list of Course Features. This can be done through the metadata flow in Studio, but the new feature does not have a corresponding icon. [RC example](https://ocw-draft-qa.global.ssl.fastly.net/courses/16-412j-cognitive-robotics-spring-2016/)

image

The insights team voted for "[CoPresent](https://mui.com/components/material-icons/?selected=CoPresent)" as the icon for launch.

image

### Expected Behavior
Icon should display next to the "Instructor Insights" course feature.

### Actual Behavior
N/A

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Close #510 

#### What's this PR do?
Adds CoPresent icon for course features.

#### How should this be manually tested?
Verify proper icon shows up in learning resource type

#### Any background context you want to provide?
The CoPresent Icon was not available in out current icon library so I had top import the google material design font icon library and used the icon from it.
 
#### Screenshots (if appropriate)
Screenshot 2022-03-10 at 7 46 38 PM

### CODE REVIEW

**Comment by dseaton** [comment]
Looks good from a user perspective!

**Review by alicewriteswrongs — CHANGES_REQUESTED** [review]
this works great!

one thing though - if we're going to add another font, could we add it in the same way that the other icon fonts are already added? Check out the config in `webpack.common.js` and the font files saved in `base-theme/assets/fonts`. It's possible also if we just update the files in there that the `co_present` icon will be available for us, it looks like @gumaerc checked those files in to the repo last April.

File: base-theme/assets/css/main.scss [hunk 1]
```diff
@@ -19,6 +19,30 @@ a.coming-soon {
 
 /* CUSTOM STYLES */
 
+@mixin icon-styles {
```
Comment (alicewriteswrongs) [inline base-theme/assets/css/main.scss:22]: nice touch with the mixin

File: base-theme/layouts/partials/head.html [hunk 2]
```diff
@@ -29,6 +29,14 @@
       url({{ partial "webpack_url.html" "/static/fonts/MaterialIcons-Regular.woff" }}) format('woff'),
       url({{ partial "webpack_url.html" "/static/fonts/MaterialIcons-Regular.ttf" }}) format('truetype');
   }
+  @font-face {
+    font-family: 'Material Icons Round';
+    font-style: normal;
+    font-weight: 400;
+    src: local("MaterialIcons-Round"),
+    url({{ partial "webpack_url.html" "/static/fonts/MaterialIconsRound-Regular.woff2" }}) format('woff2');
+
```
Comment (alicewriteswrongs) [inline base-theme/layouts/partials/head.html:38]: stray blank line here

**Review by alicewriteswrongs — APPROVED** [review]
LGTM! saw one small thing (blank line) you might want to address, feel free to merge after doing so

File: base-theme/layouts/partials/head.html (hunk 2 above)
Comment (Wassaf-Shahzad) [reply base-theme/layouts/partials/head.html]: done

**Pull request merged**